help.go

```diff
@@ -29,10 +31,11 @@ Flags:
 
 Subcommands:
   %[1]s layout - Lists available layout engine options with short help
-  %[1]s layout [layout name] - Display long help for a particular layout engine
+  %[1]s layout [name] - Display long help for a particular layout engine
+  %[1]s fmt file.d2 - Autoformat file.d2
```

Comment:
should we just call it Format? Like, in the IDE plugins, it's auto, but in the CLI the user is just... formatting

Reply:
true, will change